cookbook/docs/index.rst

```diff
@@ -12,132 +12,8 @@ Flyte to address a specific problem. Code for all of the examples in the user gu
-
-      .. prompt:: bash
-
-         python cookbook/core/flyte_basics/hello_world.py
-
-      .. raw:: html
-
-          <details>
-          <summary><a>Expected Output</a></summary>
-
-      .. prompt::
-
-         Running my_wf() hello world
-
-      .. raw:: html
-
-          </details>
-
-      .. admonition:: 🎉 **Congratulations** 🎉
-
-         You have just run your first workflow. Now, let's run it on the `sandbox cluster deployed earlier <https://docs.flyte.org/en/latest/getting_started.html>`_.
-
-   #. We've packaged all the required components to run a sandboxed flyte cluster into a single docker image. You can start
-      one by running:
-
-      .. prompt:: bash
-
-         flytectl sandbox start --source ${PWD}
-
-      .. tip::
-         In case make start throws any error please refer to the troubleshooting guide here `Troubleshoot <https://docs.flyte.org/en/latest/community/troubleshoot.html>`_
-
-      Check status:
-
-      .. prompt:: bash
-
-         flytectl sandbox status
-
-      Teardown:
-
-      .. prompt:: bash
-
-         flytectl sandbox teardown
-
-   #. Take a minute to explore Flyte Console through the provided URL.
-
-      .. figure:: https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/index/first_run_console_2.gif
-         :alt: A quick visual tour for launching your first Workflow.
-
-         A quick visual tour for launching your first Workflow.
-
-   #. Register all examples from cookbook/core into the cluster. This step compiles your python code into the intermediate
-      flyteIdl language and store them on the control plane running inside the cluster.
-
-      .. prompt:: bash
-
-         REGISTRY=cr.flyte.org/flyteorg make fast_register
-
-      .. note::
-         If the images are to be re-built, run ``make register`` command.
-
-   #. Let's launch our first execution from the UI. Visit `the console <http://localhost:30081/console/projects/flytesnacks/domains/development/workflows/core.flyte_basics.hello_world.my_wf>`_, click launch.
-
-   #. Give it a minute and once it's done, check out "Inputs/Outputs" on the top right corner to see your greeting.
-
-      .. figure:: https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/index/exercise.gif
-         :alt: A quick visual tour for launching a workflow and checking the outputs when they're done.
-
-         A quick visual tour for launching a workflow and checking the outputs when they're done.
-
-   .. admonition:: Recap
-
-     You have successfully:
-
-     #. Run a flyte workflow locally,
-     #. Run a flyte sandbox cluster,
-     #. Run a flyte workflow on a cluster.
-
-     .. rubric:: 🎉 Congratulations, now you can interactively explore Flyte's features outlined in the :ref:`Table of Contents` 🎉
-
+documenting and contributing samples easy. If this is your first time running these examples, follow the
+:ref:`environment setup guide <env_setup>` to get started.
```

Comment:
where is this?

Reply:
it's a new file

Reply:
https://github.com/flyteorg/flytesnacks/pull/721/files#diff-ac47681360eb88338688304efea94de5999015ea2a060c4f45b0dbed6773b7d7